contract/contracts/expense-account/ExpenseAccountEIP712.sol

```diff
@@ -43,7 +113,11 @@ contract ExpenseAccountEIP712 is
 
     error UnauthorizedAccess(address expected, address received);
 
-    error AuthorizedAmountExceeded(uint256 amount);
+    error AmountPerPeriodExceeded(uint256 amount);
+
+    error AmountPerTransactionExceeded(uint256 amount);
+
+    //error InvalidBudgetType();
```

Comment:
is there any intention for this comment?

Reply:
I'll remove that one, I think I had used it before but then realised it wasn't necessary